tests/flow/test_memory_usage.py

```
@@ -0,0 +1,293 @@
+        """test error reporting from invalid calls to GRAPH.MEMORY USAGE"""
+
+        # usage:
+        # GRAPH.MEMORY USAGE <GRAPH_ID> [SAMPLES <count>]
+        
+        # wrong arity
+        cmd = "GRAPH.MEMORY"
+
+        try:
+            res = self.conn.execute_command(cmd)
+            self.env.assertTrue(False)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)